tests/main.js

```diff
@@ -402,8 +402,8 @@ testBundlers(
     t.deepEqual(
       functions,
       [
-        { name: 'test', mainFile: 'test.js', runtime: 'js', extension: '.js' },
         { name: 'test', mainFile: 'test.zip', runtime: 'js', extension: '.zip' },
+        { name: 'test', mainFile: 'test.js', runtime: 'js', extension: '.js' },
```

Comment:
The reason for this changing is that we're now prioritising `.js` files over `.zip` files, which was the existing behaviour before #364.